pkgs/uefi-firmware/default.nix

```diff
@@ -73,28 +73,21 @@ let
     src = fetchFromGitHub {
       owner = "NVIDIA";
       repo = "edk2-nvidia";
-      rev = "8444db349648a77ed8e2e3047a93004c9cadb2d3"; # Latest on r35.4.1-updates as of 2023-08-07
-      sha256 = "sha256-jHyyg5Ywg/tQg39oY1EwHPBjUTE7r7C9q0HO1vqCL6s=";
+      rev = "a5ac12d729f610035f16013482fa70284f75ddfd"; # Latest on r35.5.0-updates as of 2024-03-08
+      hash = "sha256-3llK5lY/V8JZha0N8bYsK4OLd52BSMo3DtyEf8+LZ34=";
     };
     patches = edk2NvidiaPatches ++ [
-      (fetchpatch {
-        # https://github.com/NVIDIA/edk2-nvidia/pull/68
-        name = "fix-disabled-serial.patch";
-        url = "https://github.com/NVIDIA/edk2-nvidia/commit/9604259b0d11c049f6a3eb5365a3ae10cfb9e6d9.patch";
-        hash = "sha256-v/WEwcSNjBXeN0eXVzzl31dn6mq78wIm0u5lW1jGcdE=";
-      })
       # Fix Eqos driver to use correct TX clock name
       # PR: https://github.com/NVIDIA/edk2-nvidia/pull/76
-      (fetchpatch {
-        url = "https://github.com/NVIDIA/edk2-nvidia/commit/26f50dc3f0f041d20352d1656851c77f43c7238e.patch";
-        hash = "sha256-cc+eGLFHZ6JQQix1VWe/UOkGunAzPb8jM9SXa9ScIn8=";
-      })
+      # Can't find commit from Nvidia that actually fixes PR#76
```

Comment:
Want to remove this new comment? They still haven't fixed it upstream. If you need to vendor the patch like you've done, that's fine, but could you also include the commit message?

Reply:
Comment removed and commit messaged added to patch. Thanks.